build/distributions/CentOS/7/x86_64/DVD/7-simp_pkglist.txt

```diff
@@ -580,6 +592,9 @@ libXxf86dga
 libXxf86misc
 libXxf86vm
 libXxf86vm-devel
+vorbis-tools
+cdparanoia
+
 libacl
```

Comment:
These should be in alphabetical order like the rest of the packages, and without blank lines.